agents/jade/setup.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name='jade_agent',
+    version='0.1.0',
+    description='Using Blockstream Jade as hardware SSH agent',
+    author='Jamie C. Driver',
+    author_email='jamie@blockstream.com',
+    url='http://github.com/romanz/trezor-agent',
+    scripts=['jade_agent.py'],
+    install_requires=[
+        # FIXME: will need libagent version that includes jade support - 0.14.5 ??
```

Comment:
Let's fix it after this PR is merged :)

Reply:
Sure.  Seems a little circular, so maybe it needs a couple of passes.  Whatever you deem best is good with me.